### DISCUSSION TITLE
Welcome to Peridynamics.jl Discussions!

### DISCUSSION THREAD

**Discussion by kaipartmann** [kaipartmann/Peridynamics.jl#27] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DrHybris** [comment]
Hi, I would like to study the comparison between using FEM and PeriDynamics for modeling stress and displacement in a planetary gearbox. Afterwards, I want to study the wave propagation of a crack inside the gear teeth. I'm not sure if this can be done with this Julia package — if anyone can answer, I’d really appreciate it.

**Reply by kaipartmann** [reply]
Hi @DrHybris, this sounds like a interesting but also challenging task! I think it should be possible, depending on a few factors. Do you want to make dynamic simulations with explicit time integration? And how detailed is your geometry? Currently only multithreading is possible with contact simulations. And also the implemented contact model is very basic and has no friction.

**Reply by DrHybris** [reply]
Thanks for the feedback.
I didn’t know what type of dynamic simulation to use—if you have any ideas, please let me know. I’m relatively new to this.
My specialization is machine diagnosis using vibration signals (acoustic emission or acceleration), and I would like to know if it’s possible to conduct a study to determine what type of acoustic emission signal is generated based on the size of a crack.
The objective would be to predict the crack size from the actual acoustic emission data collected in real-world conditions.

**Reply by DrHybris** [reply]
Perhaps it’s a very ambitious task; even so, if it’s possible to model crack growth in a planetary gearbox based on the contact between two gears, it would be fantastic.
